discovery-provider/src/tasks/index_user_bank.py

```diff
@@ -345,12 +345,18 @@ def index_purchase(
     timestamp: datetime,
     tx_sig: str,
 ):
+    # Detect "pay extra" amount (difference between sender's balance change and price
+    # of the content)
+    extra_amount = max(
+        0, -(balance_changes[sender_account]["change"]) - purchase_metadata["price"]
```

Comment:
isn't `purchase_metadata["price"]` in cents and `balance_changes[sender_account]["change"]` in wei? Or am I misremembering?

Reply:
In track metadata it is, but we [convert it when indexing purchases](https://github.com/AudiusProject/audius-protocol/blob/6b76e070676e8c45141f491286e78e925ce256f5/discovery-provider/src/tasks/index_user_bank.py#L294).

There is another place above in this file where we do some math with this field and the `change` amount to ensure the sender paid full price for the track